catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -209,9 +207,15 @@ public OpenSearchSource(
    * called for each property specified in the metatype.xml file.
    */
   public void init() {
-    factory = createClientFactory(endpointUrl.getResolvedString(), username, password);
     configureXmlInputFactory();
-    isInitialized = true;
+  }
+
+  // lazy init
+  protected SecureCxfClientFactory<OpenSearch> getClientFactory() {
```

Comment:
✏️ Should this be `@VisibleForTesting`? (or, not overridden in the unit tests? -- see comment below)

Reply:
Fixed